src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIRequestSmelter.java

```diff
@@ -38,6 +42,14 @@
      */
     private static final double BASE_XP_GAIN = 5;
 
+    static
```

Comment:
I generally do not like the way this is done!
Move all of these to a special init class and simply call it during init.

Using static code blocks like this is a very very very very bad design!

Reply:
Okay, I resolved the below convos, because it's related to this for tracking.
My goal of this was to allow two things
a) Simplicity, if I create a trigger for a class I don't want to pass through a bunch of different classes to set it up. I want it all at one place.
b) If we move this all into an extra class we will have one huge class which has tons of imports but is quite confusing.
If you know a better way how we can handle this within a class I'm all in, but I believe we should store this in the class where we create the trigger also because of c)
c) Easy access to all values within the class without requiring to pull all of them in an interface